modules/ROOT/pages/rest-sdk/configure-operations.adoc

```diff
@@ -81,8 +80,7 @@ endpoints:
       ignored: false
 ----
 
-In this example, only the operations under the `/flights/{ID}` path will be generated in the connector,
-all other operations and triggers won't.
+In this example, only the operations and triggers under the `/flights/{ID}` path will be generated.
```

Comment:
@sathya0 I would revert this change, as matching that path won't be generated, only the operations will.
Triggers can bu unignored in a different section.